### DISCUSSION TITLE
boto3 error

### DISCUSSION THREAD

**Discussion by shivatechm** [StackStorm/st2#5478] [Q&A]
Getting below error
 File "/opt/stackstorm/packs/adms/actions/install_apache.py", line 6, in <module>
    import boto3
ModuleNotFoundError: No module named 'boto3'

**Comment by amanda11** [comment]
This looks to be a problem with your specific pack adms, and not with ST2 itself. If your pack requires python libraries such as boto3 then you will need to include them in the requirements.txt of your pack: https://docs.stackstorm.com/reference/packs.html#anatomy-of-a-pack
